src/libm/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O3 -DNDEBUG")
```

Comment:
Ok, but do we not want the highest optimisation flag at all times? Should the default build type be RelWithDebInfo then?